### PR TITLE
Fix Azure E2E tests due to breaking change in az output.

### PR DESCRIPTION
**Pull request by artursouza**
# Description

Fix Azure E2E tests due to breaking change in az output.

## Issue reference

Please reference the issue this PR will close: endgame

## Checklist

Please make sure you've  completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Unit tests passing
* [ ] End-to-end tests passing
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Specification has been updated / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_
* [ ] Provided sample for the feature / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Comment by artursouza** [comment]
I tested this locally. Could repro the issue and confirm the fix. Can only be validated here after merging.

**Comment by artursouza** [comment]
For some reason, the output is now quoted, which breaks the login for the 2nd command.

File: .github/workflows/dapr-test.yml [hunk 1]
```diff
@@ -287,7 +287,7 @@ jobs:
       - name: Login to cache registry for Windows
         if: runner.os == 'Windows' && env.CHECKOUT_REPO != '' && env.DAPR_CACHE_REGISTRY != ''
         run: |
-          $accessToken = (az acr login  -n ${{ env.DAPR_CACHE_REGISTRY }} --expose-token --query accessToken)
+          $accessToken = (az acr login  -n ${{ env.DAPR_CACHE_REGISTRY }} --expose-token --query accessToken) -replace '"'
```
Comment (ItalyPaleAle) [inline .github/workflows/dapr-test.yml:290]:
```suggestion
          $accessToken = (az acr login  -n ${{ env.DAPR_CACHE_REGISTRY }} --expose-token --query accessToken --output tsv)
```

File: .github/workflows/dapr-test.yml [hunk 2]
```diff
@@ -335,7 +335,7 @@ jobs:
       - name: Login to Azure Container Registry for Windows
         if: runner.os == 'Windows' && env.TEST_PREFIX != ''
         run: |
-          $accessToken = (az acr login  -n ${{ env.DAPR_REGISTRY }} --expose-token --query accessToken)
+          $accessToken = (az acr login  -n ${{ env.DAPR_REGISTRY }} --expose-token --query accessToken) -replace '"'
```
Comment (ItalyPaleAle) [inline .github/workflows/dapr-test.yml:338]:
```suggestion
          $accessToken = (az acr login  -n ${{ env.DAPR_CACHE_REGISTRY }} --expose-token --query accessToken --output tsv)
```

**Review by ItalyPaleAle — CHANGES_REQUESTED** [review]
See proposed changes

File: .github/workflows/dapr-test.yml (hunk 1 above)
Comment (artursouza) [reply .github/workflows/dapr-test.yml]: Done

File: .github/workflows/dapr-test.yml (hunk 2 above)
Comment (artursouza) [reply .github/workflows/dapr-test.yml]: Done.

**Comment by artursouza** [comment]
> See proposed changes

Done

**Pull request merged**